CRM/Core/BAO/Setting.php

```diff
@@ -125,8 +125,6 @@ public static function getItems(&$params, $domains = NULL, $settingsToReturn) {
   /**
    * Store an item in the setting table.
    *
-   * _setItem() is the common logic shared by setItem() and setItems().
-   *
    * @param object $value
```

Comment:
Could we remove this  ``object`` type hint? This is what made Jens cast it to object in the first place. 

I know this code needs to go anyway, but that's no reason it should be wrong/misleading while it's still in...

Reply:
Can you do a type hint in comments? I didn't realise that was possible - or do you just mean the fact it's saying the param is an object?

Reply:
I believe it would be the latter and therefore in IDEs etc it would say it is expecting an object

Reply:
OK - I think it makes sense to fix then - but I'll leave out of scope of this pr (which depends on #15981 being merged)

Reply:
That's fair. @jensschuppe could you create a separate PR for this?